styles.css

```diff
@@ -0,0 +1,132 @@
+html {
+	font-family: "Trebuchet MS", sans-sarif;
+	box-sizing: border-box;
+	font-size: 10px;
+}
+
+body {
+	background-color:#0a415c;
+	margin: 0;
+	padding: 0;
+}
+
+h2 {
+	font-size: 3rem;
+}
+
+h3 {
+	font-size: 2rem;
+}
+
+p {
+	font-size: 1.5rem;
+}
+
+nav {
+	display: flex;
+	flex-direction: row;
+	flex-wrap: nowrap;
+	justify-content: space-between;	
+	align-items: center;
+}
+
+.nav-bar {
+	width: 30%;
+	display: flex;
+	flex-direction: row;
+	flex-wrap: nowrap;
+	justify-content: space-around;
+	color: black;
+	font-size: 2rem;
+	padding: 20px;
+	color: white;
+}
+
+.nav-bar a:link, .nav-bar a:visited {
+	color: white;
+}
+.nav-bar a:hover {
+	color:#048230;
+}
+
+
+.cv-download-button {
+	width: 150px;
+	height: 30px;
+	border: 2px solid black;
+	background-color: #0a415c;
+	color: black;
+	font-size: 1.6rem;
+	cursor: pointer;
+	border-radius: 20px;
+	
+}
+
+.cv-download-button {
```

Comment:
these style lines conflict with the lined above. Consolidate and choose which you want to keep.